### PR TITLE
added problem from leetcode

### PR DESCRIPTION
**Pull request by Jay-Shah5109**
added new code for longest pallindromic string

### CODE REVIEW

File: PallindromicString.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        for (int i = 0 ; i < s.length()-1 ; i++) {
+            // for odd numbered length of string
+            findPallindromeLengthFromMiddleIndex(s, i, i);
+            // for even string length
+            findPallindromeLengthFromMiddleIndex(s, i, i + 1);
+        }
+        return s.substring(resultStart, resultStart + resultLength);
+    }
+    public static void main(String[] args) {
+        String  s = "xyzabcdfggfdcbazyz";
```
Comment (Jay-Shah5109) [inline PallindromicString.java:27]: test for all possible edge cases
